frontend/test/jest-setup-env.js

```diff
@@ -2,4 +2,12 @@ import fetchMock from "fetch-mock";
 
 beforeEach(() => {
   fetchMock.restore();
+  fetchMock.catch((url, request) => {
+    const errorMessage = `Caught not mocked ${request.method} request to url: ${url}`;
```

Comment:
```suggestion
    const errorMessage = `Caught unmocked ${request.method} request to: ${url}`;
```

Reply:
I used unmocked initially, but later on I found myself thinking it's something that was mocked before and unmocked later. If you think `unmocked` is better, I'll change it for sure 👍

Reply:
> I found myself thinking it's something that was mocked before and unmocked later.

Good point. Let's keep it unambiguous :+1:

(although native English speaker @oisincoveney didn't see a problem with my suggestion :smile: )

Reply:
I think it _should_ be ok, since if the dev is getting this message then they'll definitely see that the request was "unmockeded" somehow even if they tried to mock it before